tests/Integration/Behaviour/Features/Scenario/Product/update_product_type.feature

```diff
@@ -1,4 +1,7 @@
 # ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags update-product-type
+@restore-all-tables-before-feature
+@reboot-kernel-before-feature
+@clear-cache-before-feature
```

Comment:
```suggestion
```
This behat scenario already has a restore policy based on `@restore-products-before-feature` so restoring the tables twice will only make the test longer for no reason

As for the clear cache and kernel reboot before it shouldn't be its responsibility, it didn't need it previously so if something has been messed in the cache and kernel services it means it's an impact from another feature executed before So I guess it comes from `update_prices_without_states.feature` so it's the responsibility of the previous feature to clean correctly after its execution

If we leave the cleaning to the following one it means the next time we add another feature in between (because of alphabetical order) then it will be this new feature which will need to clean And on some environment the alphabetical order is not strictly the same so it becomes even more a nightmare All the more reason to leave the cleaning to the one responsible for it